src/apis/modTools.es6.js

```diff
@@ -1,19 +1,34 @@
 import apiRequest from '../apiBase/apiRequest';
 
+const DISTINGUISH_TYPE = {
+  'moderator': 'yes',
+  '': 'no',
+  'admin': 'admin',
+};
+
```

Comment:
This feels a little weird. Would rather do something like:
```
{
  MODERATOR: 'yes',
  NONE: 'no'
}
```

But in the client app we are passing around `'moderator'` and `''` strings (and passing those strings to node-api-client). Maybe there is some special javascript syntax I can use?